app/assets/javascripts/angular/controllers/auth/profile.js.erb

```diff
@@ -0,0 +1,27 @@
+app.controller('ProfileCtrl', ['$scope', '$http', '$location', '$auth', function($scope, $http, $location, $auth){
+
+  $scope.message = "Public Profile";
+
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+  $auth.validateUser()
+    .then(function(response) {
+      $scope.loggedIn = true;
+      $http.get(url + '/users/' + response.id).success(function(response) {
+        
+        $scope.profile = response;
+        
+        if (!response.image) {
+          $scope.profile.image = "https://lh3.googleusercontent.com/-hoI3rXz7oy0/AAAAAAAAAAI/AAAAAAAAAAA/UBf8PfFVAsQ/photo.jpg"
```

Comment:
Add the image to your `assets/images` and then you can use [image_url](http://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html] to generate the link:

```js
$scope.profile.image = "<%= image_url("rails.png") %>"
```

imagine if googleusercontent.com goes down... you'll loose access to the image no?